XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -858,9 +858,6 @@
   <data name="POOL_AUTH_ENABLE_FAILED_DUPLICATE_HOSTNAME" xml:space="preserve">
     <value>Failed to enable external authentication, a duplicate hostname was detected.</value>
   </data>
-  <data name="POOL_AUTH_ENABLE_FAILED_INVALID_ACCOUNT" xml:space="preserve">

```

Comment:
We need to keep the instance in line 876 as it was added to fix CA-254480 (https://github.com/xenserver/xenadmin/commit/04cfd02bd0fce3e46f2a99f82bdfd7167ecce663). I'll move it here instead.